newIDE/app/src/ProjectManager/index.js

```diff
@@ -738,6 +749,27 @@ export default class ProjectManager extends React.Component<Props, State> {
                 )
             }
           />
+          {!!this.state.layoutVariablesDialogOpen && (
+          <VariablesEditorDialog
+            open={this.state.layoutVariablesDialogOpen}
+            variablesContainer={project.getVariables()}
```

Comment:
You're passing the project variables, so the the dialog will show the global variables :/

Reply:
@4ian Changed it to show the layout variables now. 
![Screenshot (82)](https://user-images.githubusercontent.com/32250849/76690504-d07cc780-6640-11ea-935b-ec63f355951f.png)